### DISCUSSION TITLE
Logs should expand across screen, "Update" should only update and add a "Down" button

### DISCUSSION THREAD

**Discussion by nickshanks347** [louislam/dockge#102] [Feature Request]
So not sure if this is possible but it may be useful to allow users to change around the three main elements on a stack screen: Containers, Terminal and docker-compose.yml so they can organize the layout however they want. Otherwise I think Terminal should be above Containers and docker-compose but have it span the whole width of the screen. Length can stay at ~15-20 lines. 

Also, the "Update" button should only pull the containers, not restart them. There may be times when this could be useful and could be turned on/off with a switch in Settings. 

Maybe also add a "Down" button which would run `docker compose down`.

**Comment by eded333** [comment]
https://github.com/louislam/dockge/discussions/437#discussioncomment-8732500

Maybe this could help for the log view.
